### PR TITLE
GCI- 142 gci-tixpro removed some deprecated code

### PR DESCRIPTION
**Pull request by ineshtickoo**
getComment() replaced by getComments()
setComment() replaced by setComments()

## Issue(s) I worked on

see 
1) https://sonar.openmrs.org/project/issues?id=org.openmrs%3Aopenmrs&open=AW5r7d48X_1wboXLs5bE&resolved=false
2) https://sonar.openmrs.org/project/issues?id=org.openmrs%3Aopenmrs&open=AW5r7d48X_1wboXLs5bF&resolved=false

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you include the ticket id in your commit message as recommended at? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by dkayiwa** [comment]
@ineshtickoo i see three pull requests from you that look the same. Should we close the duplicates?

File: api/src/main/java/org/openmrs/Allergy.java [hunk 1]
```diff
@@ -183,23 +183,6 @@ public void setSeverity(Concept severity) {
...
-	}
-	
-	/**
-	 * @param comment the comment to set
-	 * @deprecated as of 2.3.0, replaced by {@link #setComments(String)}
-	 */
-	@Deprecated
-	public void setComment(String comment) {
-		setComments(comment);
-	}
```
Comment (PrathameshMutkure) [inline api/src/main/java/org/openmrs/Allergy.java]: Hii @ineshtickoo, kindly fix the travis-ci build failure

As per my observation, you forgot to update the method names in test files.

**Comment by dkayiwa** [comment]
Closing this because we do not remove deprecated code until 3 releases after deprecation.

**Pull request closed without merging**